install/docker-extension/Makefile

```diff
@@ -53,5 +53,11 @@ ui-build:
 
 ui:
 	cd ui/src; npm run start; cd ../..;
+# Make easier to debug the UI
+link:
```

Comment:
interesting... How does this work?

Reply:
it provide you a similar development experience on the docker with all dev tools.

```bash
make ui
make link
```